Config/taste.cfg

```diff
@@ -0,0 +1,35 @@
+[gdbServer]
+address = %(TASTE_RASPI_ADDRESS)s
+userName = %(TASTE_RASPI_USER)s
+password = %(TASTE_RASPI_PASSWORD)s
+path= /usr/local/bin/openocd
+args=   -f /usr/local/share/openocd/scripts/interface/cmsis-dap.cfg \
+        -f /usr/local/share/openocd/scripts/board/atmel_samv71_xplained_ultra.cfg \
+        -f /home/pi/broadcastOpenocd.cfg
+verbose = True
+
+[gdb]
+address = %(TASTE_RASPI_ADDRESS)s:%(TASTE_RASPI_PORT)s
+path = gdb-multiarch
+verbose = True
+
+[ioConsole]
+address = %(TASTE_RASPI_ADDRESS)s
+userName = %(TASTE_RASPI_USER)s
+password = %(TASTE_RASPI_PASSWORD)s
+baudrate = 115200
+port = 5005
+path = /dev/ttyACM0
+verbose = True
+
+[ioUart4]
+address = %(TASTE_RASPI_ADDRESS)s
+userName = %(TASTE_RASPI_USER)s
+password = %(TASTE_RASPI_PASSWORD)s
+baudrate = 38400
```

Comment:
I would recommend to make it lower. Or, reconfigurable with a lower default (stable 9600 for tests, potentially higher for demos/etc)